surveys/2021-annual-survey/questions.md

```diff
@@ -33,7 +33,7 @@ Type: select one
 Type: select all that apply (optional)
 
 - I plan to return to using Rust in the future
-- I consider myself part of the Rust Community
+- I consider myself a part of the Rust community
```

Comment:
It reads better without the 'a' to me, agree on the lower case c. Doesn't matter too much either way

Reply:
I think you're right. I'll change back